packages/calcite-components/src/components/combobox/combobox.e2e.ts

```diff
@@ -1918,4 +1918,58 @@ describe("calcite-combobox", () => {
       });
     });
   });
+
+  it("should not open combobox menu with spacebar while focusing on chip's close button", async () => {
+    const page = await newE2EPage();
+    await page.setContent(html`
+      <calcite-combobox label="test" placeholder="placeholder" max-items="10" scale="m">
+        <calcite-combobox-item-group label="Pokemon">
+          <calcite-combobox-item value="Pikachu" text-label="Pikachu"></calcite-combobox-item>
+          <calcite-combobox-item value="Venusaur" text-label="Venusaur"></calcite-combobox-item>
+          <calcite-combobox-item value="Charizard" text-label="Charizard"></calcite-combobox-item>
+          <calcite-combobox-item-group label="Cutest Pokemon">
+            <calcite-combobox-item value="Bulbasaur" text-label="Bulbasaur"></calcite-combobox-item>
+            <calcite-combobox-item value="Squirtle1" text-label="Squirtle1">
+              <calcite-combobox-item value="Squirtle2" text-label="Squirtle2">
+                <calcite-combobox-item value="Squirtle3" text-label="Squirtle3">
+                  <calcite-combobox-item value="Squirtle4" text-label="Squirtle4"></calcite-combobox-item>
+                </calcite-combobox-item>
+              </calcite-combobox-item>
+            </calcite-combobox-item>
+          </calcite-combobox-item-group>
+        </calcite-combobox-item-group>
+      </calcite-combobox>
+    `);
+
+    const combobox = await page.find("calcite-combobox");
+    const openEvent = page.waitForEvent("calciteComboboxOpen");
+    await combobox.click();
+    await openEvent;
+
+    const item1 = await combobox.find("calcite-combobox-item[value=Pikachu]");
+    await item1.click();
+    const item2 = await combobox.find("calcite-combobox-item[value=Charizard]");
+    await item2.click();
+    const item3 = await combobox.find("calcite-combobox-item[value=Squirtle3]");
+    await item3.click();
+
```

Comment:
No Bulbasaur love? 😭

![](http://haruhichan.com/wp-content/uploads/Which-Gen1-Pokemon-Starter-Is-The-Best-Bulbasaur.gif)